src/Datadog.Trace.ClrProfiler.Managed/ScopeFactory.cs

```diff
@@ -191,6 +192,48 @@ public static Scope CreateDbCommandScope(Tracer tracer, IDbCommand command)
             return scope;
         }
 
+        public static Scope CreateRabbitMQScope(Tracer tracer, out RabbitMQTags tags, string command, ISpanContext parentContext = null, string spanKind = SpanKinds.Client, DateTimeOffset? startTime = null, string queue = null, string exchange = null, string routingKey = null)
```

Comment:
Can we make `spanKind` not optional? I can see this leading to subtle bugs because we forgot to specify the span kind somewhere.

Reply:
Yeah you're right, I'll make this a required parameter

Reply:
Resolved in 26f8024013e710203be90c3e9143cc6eb8e13d77